handlers.go

```diff
@@ -66,6 +66,9 @@ func AddBeer(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	if err != nil {
 		http.Error(w, "Bad beer - this will be a HTTP status code soon!", http.StatusBadRequest)
 		return
+	} else {
```

Comment:
Whoops, haha, good catch! Can you remove the else block though and put it below the if statement? The if err != nil will return if there's an error, so no need for the else here :)